api/PaymentRequest.json

```diff
@@ -478,13 +478,7 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": false,
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.payments.request.enabled"
-                }
-              ]
+              "version_added": false
```

Comment:
Should this be version 55 with the flag? That would be consistent with Firefox support for other features in this file.

Reply:
Ah, it does look like Firefox supports the option, as per https://searchfox.org/mozilla-central/rev/66b43502bdbe4b62cbc8b0c8ff10efc5707352e4/dom/webidl/PaymentRequest.webidl#124.